src/tooltip.tsx

```diff
@@ -478,12 +454,12 @@ const Tooltip: React.FC<Props> = ({children, description, target, title, targetL
                             <div className={classes.arrow} />
                         </div>
                         {(title || description) && (
-                            <>
-                                {title && <p className={classes.title}>{title}</p>}
-                                {description && <p className={classes.description}>{description}</p>}
-                            </>
+                            <Stack space={4}>
+                                {title && <Text2 medium>{title}</Text2>}
+                                {description && <Text2 regular>{description}</Text2>}
+                            </Stack>
                         )}
-                        {children}
+                        {extra || children}
```

Comment:
why do we need `extra`? just use `children`

Reply:
looking for feature parity

Reply:
it doesn't make sense to have both, if we add extra we should remove children. But removing children is a breaking change, so if we prefer extra, then mark children as deprecated